packages/next/pages/_document.js

```diff
@@ -117,14 +117,15 @@ export class Head extends Component {
         }
         return child
       })
+      if (this.props.crossOrigin) console.warn('<Head crossOrigin=""> is deprecated, use next.config.js option: crossOrigin')
```

Comment:
Let's add an err.sh link here. Which basically works like `err.sh/zeit/next.js/head-crossorigin-deprecated` which will be redirected to this repo's `errors/head-crossorigin-deprecated.md` file